make-intranet-public/README.md

```diff
@@ -0,0 +1,63 @@
+# Making parts of the intranet public
+
+## Status quo
+
+We currently have an internal `hugo` page which we use for internal documentation.
+This page is called `intranet` and contains all kinds of documentation around our operational work as well as security and development.
+The source code for our `intranet` is in a private GitHub repository which contains other security and business relevant data.
+
+## Goal
+
+The non-security relevant parts of the intranet should be public.
+Making them public enables us to collaborate with other companies more easily.
+It additionally allows us to enable outside contributors to work on projects more independently.
+
+A very direct (and most urgent) goal is to enable outside collaborators quickly who need to work on docs for us.
+This is mainly related to companies who will help us implement features in our managed apps but will not get full access to our private docs.
+
+## Considerations
+
+Content in the `intranet` is not fit for our general customer base.
+It does not fit into our general product documentation but some of the content may be interesting to them.
+I will not specify which parts of the current intranet may be better suited for our product documentation,
+that is a decision which can be made independently of this proposal.
+
+## Proposed solution
+
+We create a new public repository called `handbook`.
+This repository uses the same general structure and hugo setup as our intranet.
+
+A similar search setup to our docs should be setup which covers the `handbook` as well as the `intranet`.
+That search will then be accessible through the `intranet`, linking to both sources of truth.
+Therefore the `intranet` can remain as the single source of truth for Giant Swarm employees to aggregate information.
+Most importantly, it becomes trivial to link URLs from either source as search results will forward you to the specific article.
+
+The `handbook` repository itself should also be setup with `hugo` to host its content under `handbook.giantswarm.io` making it easily accessible for outside collaborators.
+
+The complete setup would then look as follows:
+- `handbook` only contains information we are comfortable with having public.
+- `intranet` only contains truly private information but imports all content from `handbook` into a unified search.
+- `docs` contains product documentation which is directly linked on our website as before.
+
+### Transitioning content into the public handbook
+
+Content will be transitioned into the public handbook on a "if-needed" basis for now.
+We have some usecases which require our internal docs to be public (e.g. relevant docs to external collaborators, people related docs which help with hiring, ... ).
+
+Once the new concept is proven with the currently needed docs being public, then a `public by default` policy should be put in place.
+This should then by followed by all non-critical docs being moved to the public repo in quick succession.
```

Comment:
Should we decide a initial structure that will help ppl to move content?

Example

-> Operations
    -> Operation model
    -> Run books
    -> ...

-> Organization structure
    -> Chapters
    -> SIGs
    -> ...
    
-> ...
    
I think it will be useful if we put some effort into drawing an outline instead of leaving ppl to classify content freely

Reply:
I like this idea.